packages/fxa-auth-server/lib/senders/templates/layouts/subscription.html

```diff
@@ -1,125 +1,357 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" lang="{{language}}" xml:lang="{{language}}">
+<!DOCTYPE html>
```

Comment:
More brain-hurty email HTML, but also based on the template from Marketing Cloud exports